configure

```diff
@@ -20,9 +20,50 @@ else
   EXTRA_PKG_LIBS=-latomic
 fi
 
+CXX=$("${R_HOME}"/bin/R CMD config CXX)
```

Comment:
On some platforms, the `/bin/sh` does not support `$()` for running commands. You need to use backticks instead. I recently ran into this on Solaris.